ros_buildfarm/common.py

```diff
@@ -267,16 +267,19 @@ def get_short_os_name(os_name):
 
 def get_short_os_code_name(os_code_name):
     os_code_name_mappings = {
-        'jessie': 'J',
         'saucy': 'S',
-        'stretch': 'S',
         'trusty': 'T',
         'utopic': 'U',
         'vivid': 'V',
         'wily': 'W',
         'xenial': 'X',
         'yakkety': 'Y',
         'zesty': 'Z',
+        'artful', : 'A',
```

Comment:
Syntax error of extra comma on this and the next two lines.

Reply:
Ouch, good call.  Fixed in a85cc40179feaec881aacf0d2dd4d32e488f1d0d